packages/core/core-flows/src/cart/workflows/complete-cart.ts

```diff
@@ -34,36 +31,54 @@ export type CompleteCartWorkflowInput = {
   id: string
 }
 
+export const THREE_DAYS = 60 * 60 * 24 * 3
+
 export const completeCartWorkflowId = "complete-cart"
 /**
  * This workflow completes a cart.
  */
 export const completeCartWorkflow = createWorkflow(
-  completeCartWorkflowId,
+  {
```

Comment:
The only things that changed in the file was:
- Workflow configuration
- Check if order exists before completing cart

The rest is indentation as a result of the `when-then` addition